unsloth_zoo/vision_utils.py

```diff
@@ -54,21 +54,39 @@
     "<|IMG_PATCH|>",      # Cohere
 ]
 
+from __future__ import annotations
+
 import torch
 from PIL import Image
 import base64
 from io import BytesIO
 import math
+import time
+import sys 
+import warnings
+import os
+import logging
+import copy
+from functools import lru_cache
+
+logger = logging.getLogger(__name__)
+
 import requests
-from typing import Union, Tuple, List, Dict
+import torchvision
+from packaging import version
+from typing import Optional, Union, Tuple, List, Dict
+from torchvision import io, transforms
+from torchvision.transforms import InterpolationMode
+
 IMAGE_FACTOR = 28
 MIN_PIXELS = 4 * 28 * 28
 MAX_PIXELS = 16384 * 28 * 28
 MAX_RATIO = 200
 
 VIDEO_MIN_PIXELS = 128 * 28 * 28
 VIDEO_MAX_PIXELS = 768 * 28 * 28
-VIDEO_TOTAL_PIXELS = 24576 * 28 * 28
+VIDEO_TOTAL_PIXELS = int(float(os.environ.get('VIDEO_MAX_PIXELS', 128000 * 28 * 28 * 0.9)))
+logger.info(f"set VIDEO_TOTAL_PIXELS: {VIDEO_TOTAL_PIXELS}")
```

Comment:
Using the logger is fine, but we generally want to keep logging less cluttered unless the user wants it. In other parts of the library we check for `do_logging = os.environ.get("UNSLOTH_ENABLE_LOGGING", "0") == "1"` to determine whether or not to log. If you'd like to stick with logger I'd suggest configuring the logging level based on the env variable, and no prints/logs if the variable is off, and prints if the variable is on. Best to also prepend "Unsloth: " before the actual string to log.